### PR TITLE
added TestTeamLogin

### PR DESCRIPTION
**Pull request by rodenbe4**
added the following test cases:
-log in team1
-log in team2, while team1 is logged in
-log out team1
-log out team2; team2 is not currently logged in

### CODE REVIEW

File: team.py [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+    def setTeamName(self,n):
+        pass
+    def setTeamUserName(self,u):
+        pass
+    def setTeamPassword(self,p):
+        pass
+    def login(self,username,password):
+        pass
+    def logout(self):
+        pass
```
Comment (Mandabee) [inline team.py]: add:
def displayMenu(self):
     pass

File: team.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import user
+
+class Team(user.User):
+    def __init__(self,n):
+        self.name = n
+        self.username = "team"
```
Comment (tekkeon) [inline team.py]: Do we want these generic values at all or should they just not be anything to start with? Or possibly we could have the init method just take in a name, username, and password...?

File: user.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import abc
```
Comment (tekkeon) [inline user.py]: I thought I was writing the user interface but alright haha

File: tests/ut_team.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import team
+import unittest
+
+class TestTeamLogin(unittest.TestCase):
+    def setUp(self):
+        self.team1 = team.Team("Name1")
+        self.team2 = team.Team("Name2")
+    def test_setTeamName(self):
+        self.assertEquals("Team name changed to n1", self.team1.setTeamName("n1"), "Error: changing team name failed")
```
Comment (tekkeon) [inline tests/ut_team.py]: Should be written like this:

def test_setTeamName(self):
    self.team1.set_team_name("n1")
    self.assertEquals(self.team1, "n1", "Error Message")

Same for below tests. Error messages are printed to console, not returned by these functions.

File: user.py (hunk 3 above)
Comment (tekkeon) [inline user.py]: Also, can we have a naming convention for interfaces like interface_user.py or user_interface.py just so we can distinguish them from each other?

**Pull request merged**